zerver/webhooks/harbor/view.py

```diff
@@ -64,12 +56,14 @@ def handle_scanning_completed_event(
     payload: Dict[str, Any], user_profile: UserProfile, operator_username: str
 ) -> str:
     scan_results = ""
-    scan_summaries = payload["event_data"]["resources"][0]["scan_overview"]["components"]["summary"]
-    summaries_sorted = sorted(scan_summaries, key=lambda x: x["severity"], reverse=True)
-    for scan_summary in summaries_sorted:
-        scan_results += "* {}: **{}**\n".format(
-            VULNERABILITY_SEVERITY_NAME_MAP[scan_summary["severity"]], scan_summary["count"]
-        )
+    scan_summaries = payload["event_data"]["resources"][0]["scan_overview"][
+        "application/vnd.security.vulnerability.report; version=1.1"
+    ]["summary"]["summary"]
```

Comment:
Could you confirm that `"application/vnd.security.vulnerability.report; version=1.1"` is the only possibility here? It seems that [this script](https://github.com/goharbor/harbor/blob/9afe596403e83403024ab40d3c138f88cf4661c2/tests/apitests/python/library/artifact.py#L9-L12) suggests otherwise where it includes `"application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0"` (probably a format for their legacy scanner). Getting a fixture for that format is not necessary but we should have error handling here if this key does not present in the payload.

Reply:
Noted. Given that we definitely need `"application/vnd.security.vulnerability.report; version=1.1"` to present as we need the information inside this key, do we raise an error if this key is missing? (perhaps UnsupportedWebhookEventType?)

Reply:
Noted. Given that we definitely need `"application/vnd.security.vulnerability.report; version=1.1"` to present as we need the information inside this key, do we raise an error if this key is missing? (perhaps `UnsupportedWebhookEventType`?)

Reply:
Potentially this will be covered in the `WildValue` refactorings. But yes, for now, we can just fall back to `UnsupportedWebhookEventType`.